app/encryption.py

```diff
@@ -67,35 +66,17 @@ def sign_with_all_keys(self, to_sign: str | NotificationDictToSign) -> List[str
             signed.append(URLSafeSerializer(k).dumps(to_sign, salt=self.salt))
         return signed
 
-    def sign_with_all_dangerously_salted_keys(self, to_sign: str | NotificationDictToSign) -> List[str | bytes]:
-        """Sign a string or dict with all the individual keys in the class secret key list, and the DANGEROUS_SALT.
-
-        Args:
-            to_sign (str | NotificationDictToSign): The string or dict to sign.
-
-        Returns:
-            List[str | bytes]: A list of signed values.
-        """
-        signed = []
-        for k in reversed(self.secret_key):  # reversed so that the default key is last
-            signed.append(URLSafeSerializer(k).dumps(to_sign, salt=self.dangerous_salt))
-        return signed
-
     def verify(self, to_verify: str | bytes) -> Any:
         """Checks the signature of a signed value and returns the original value.
-        Currently checks against both self.salt and self.dangerous_salt.
-        After everything is signed with the new salts, this will only check against self.salt.
+        Raises BadSignature if the signature is invalid.
```

Comment:
I thought there was a better format for documenting raises, but it's not part of the [PEP 257](https://peps.python.org/pep-0257/). There are different conventions though depending on the community and tool as described in this [stackoverflow entry](https://stackoverflow.com/questions/3898572/what-are-the-most-common-python-docstring-formats).

Reply:
Yeah, true enough, other places in the code such as [here](https://github.com/cds-snc/notification-api/blob/main/app/dao/notifications_dao.py#L63-L71) we put the Raise in its own section, I've made that change here.